vpr/src/route/connection_box_lookahead_map.cpp

```diff
@@ -498,15 +447,6 @@ float ConnectionBoxMapLookahead::get_map_cost(int from_node_ind,
 
     float expected_cost = criticality_fac * expected_delay + (1.0 - criticality_fac) * expected_congestion;
 
-    VTR_LOGV_DEBUG(f_router_debug, "Requested lookahead from node %d to %d\n", from_node_ind, to_node_ind);
```

Comment:
Please back some of these diagnostics, I've been keying off some of them

Reply:
Fixed.